pkgs/applications/misc/bottles/default.nix

```diff
@@ -7,6 +7,7 @@
 , freetype, p7zip, gamemode, mangohud
 , bottlesExtraLibraries ? pkgs: [ ] # extra packages to add to steam.run multiPkgs
 , bottlesExtraPkgs ? pkgs: [ ] # extra packages to add to steam.run targetPkgs
+, wineWow ? wineWowPackages.minimal
```

Comment:
Please move this to top-level

Reply:
Do you move in `pkgs/top-level/all-packages.nix` like this? 
``` diff
-  bottles = callPackage ../applications/misc/bottles { };
+  bottles = callPackage ../applications/misc/bottles {
+    wineWow = wineWowPackages.minimal
+  };
```

Reply:
yes :)